src/components/TextInput/TextInput.tsx

```diff
@@ -11,12 +12,14 @@ type TextInputProps = {
   isRequired?: boolean;
   name: string;
   pattern?: string;
+  spellCheck?: 'true' | 'false';
```

Comment:
Would a boolean prop be more appropriate here?

Reply:
yeap, I will update this :)

Reply:
hasSpellCheck also follows the convention we've established with other form control properties e.g. `isRequired` `isValid` etc